CONTRIBUTING: improve wording about example count limit (#5760)
tldr-pages/tldr: CONTRIBUTING.md

The basic format of a `tldr` page is a set of concrete usage examples.
Here are a few guidelines to get started:

> 1. Try to keep pages at around 5 examples. Pages can be longer if needed or shorter where that makes more sense, but 
>    specify one example at least and 8 examples at most.

Reviewer: This is better! I still think that mentioning a minimum number of examples is unwarranted here, especially since we say above: "The basic format of a `tldr` page is a set of concrete usage examples."

How about this?

```suggestion
1. Try to keep pages at around 5 examples. Pages can be longer or shorter when appropriate, but don't exceed  8 examples.
```

Author: That sounds nice. I'll commit that.

>    Remember it's OK if the page doesn't cover everything; that's what `man` is for.

Reviewer: Why remove the comma? I think it reads better with it.

Author: I probably accidentally pressed `d` in my editor or so… I wasn't intending to remove it. I'll re-add it.

Here are a few guidelines to get started:

1. Try to keep pages at around 5 examples. Pages can be longer if needed, but don't exceed 8 examples.

>    If specifying less than 5 makes more sense due to, for example, the command's scope being quite narrow, it is also 
>    possible to specify less, however, at least 1 example should always be present.

Reviewer 2: I'd put it slightly differently...
```suggestion
   If specifying less than 5 makes more sense due to, for example, the command's scope being quite narrow, it's fine to due so; however, at least 1 example should always be present.
```

Author: Yeah, that sounds a bit better indeed. More compact and such. One moment…
